tests/acceptance/extract-i18n.spec.js

```diff
@@ -0,0 +1,51 @@
+const tmp = require('../helpers/tmp');
+const ng = require('../helpers/ng');
+const existsSync = require('exists-sync');
+const path = require('path');
+const root = process.cwd();
+const expect = require('chai').expect;
+
+// TODO: Enable when tests get validated
+describe.skip('Acceptance: ng xi18n', function() {
+  beforeEach(function() {
+    this.timeout(180000);
+    return tmp.setup('./tmp').then(function() {
+      process.chdir('./tmp');
+    }).then(function() {
+      return ng(['new', 'foo', '--link-cli']);
```

Comment:
Bower should never install now, since we removed that code and those dependencies from Angular CLI.

I think this test would be better suited as an e2e test since you change files and do a lot of FS work. If you need help adding an e2e test, reach me out on gitter and/or by email. I'll help you.